test/io/testcjsonwriter.py

```diff
@@ -31,6 +31,15 @@ def test_init(self):
         # The object should keep the ccData instance passed to its constructor.
         self.assertEqual(cjson.ccdata, data)
 
+    def test_subclass(self):
+        """Is the writer a subclass of the abstract file writer?"""
+        fpath = os.path.join(__datadir__, "data/ADF/basicADF2007.01/dvb_gopt.adfout")
+        self.assertTrue(os.path.exists(fpath))
+        data = cclib.io.ccread(fpath)
+        writer = cclib.io.cjsonwriter.CJSON(data)
```

Comment:
Can you just use an empty `ccData` instance instead of reading from disk?

Reply:
Ideally, but
```
======================================================================
ERROR: test_init (test.io.testcjsonwriter.CJSONWriterTest)
Does the class initialize correctly?
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/eric/development/cclib_berquist/test/io/testcjsonwriter.py", line 28, in test_init
    cjson = cclib.io.cjsonwriter.CJSON(data)
  File "/home/eric/development/cclib_berquist/cclib/io/cjsonwriter.py", line 34, in __init__
    super(CJSON, self).__init__(ccdata, terse=terse, *args, **kwargs)
  File "/home/eric/development/cclib_berquist/cclib/io/filewriter.py", line 70, in __init__
    self.obmol, self.pbmol = self._make_openbabel_from_ccdata()
  File "/home/eric/development/cclib_berquist/cclib/io/filewriter.py", line 107, in _make_openbabel_from_ccdata
    obmol = makeopenbabel(self.ccdata.atomcoords,
AttributeError: 'ccData' object has no attribute 'atomcoords'
```

Reply:
What about an empty `ccData` instance where you add required attributes? Something like  `data.atomcoords = numpy.zeros((1,3))`)?

Reply:
I agree, it'd be better to have some kind of fixture here instead of relying on the parser working correctly.